packages/features/blacklist/operations/check-if-email-in-blacklist.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { startSpan } from "@sentry/nextjs";
+
+import type { Blacklist } from "../blacklist.model";
+import { BlacklistRepository } from "../blacklist.repository";
+
+/**
+ * Controllers use Presenters to convert the data to a UI-friendly format just before
+ * returning it to the "consumer". This helps us ship less JavaScript to the client (logic
+ * and libraries to convert the data), helps prevent leaking any sensitive properties, like
+ * emails or hashed passwords, and also helps us slim down the amount of data we're sending
+ * back to the client.
+ */
+function presenter(blacklistedEmail: Blacklist | null) {
```

Comment:
Yes it is part of the clean architecture code examples I've been following and it makes quite a lot of sense. This would help us to not worry about data transformations until last line before sending to the client. We actually follow a similar approach in v1 API but using zone schemas.

Reply:
Yeah I like it. Let's document and share with team.